miscDeps

```diff
@@ -1 +1 @@
-Subproject commit 239621041225ce15b93d6aeb96945eaf67e10898
+Subproject commit a7f918c421e91f1b1601ab822ede2c869275ef5c
```

Comment:
Oh. How did that happen?
Whoops. Does git pull modify anything because I have master set to pull 
from origin/master here locally.

On 3/30/2016 9:15 PM, James Teh wrote:
>
> In miscDeps 
> <https://github.com/nvaccess/nvda/pull/5842#discussion_r57996394>:
>
> > @@ -1 +1 @@
> > -Subproject commit 239621041225ce15b93d6aeb96945eaf67e10898
> > +Subproject commit a7f918c421e91f1b1601ab822ede2c869275ef5c
>
> This PR updates miscDeps, but it shouldn't. Just something to be 
> careful of for future commits.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub 
> <https://github.com/nvaccess/nvda/pull/5842/files/ae68ffa67b8deeae01c6d60f0c8050b4ec4d05b1#r57996394>
>

-- 
------------------------------------------------------------------------


    Derek Riemer

  * Department of computer science, third year undergraduate student.
  * Proud user of the NVDA screen reader.
  * Open source enthusiast.
  * Member of Bridge Cu
  * Avid skiier.

Websites:
Honors portfolio <http://derekriemer.com>
Awesome little hand built weather app! 
<http://django.derekriemer.com/weather/>

email me at derek.riemer@colorado.edu <mailto:derek.riemer@colorado.edu>
Phone: (303) 906-2194